src/ngraph/op/topk.cpp

```diff
@@ -443,21 +443,218 @@ void op::v1::TopK::set_k(size_t k)
         op::Constant::create(element::i64, Shape{}, {k})->output(0));
 }
 
-namespace ngraph
+// v3 version starts
+constexpr NodeTypeInfo op::v3::TopK::type_info;
+
+op::v3::TopK::TopK(const Output<Node>& data,
+                   const Output<Node>& k,
+                   const int64_t axis,
+                   const std::string& mode,
+                   const std::string& sort,
+                   const element::Type& output_indices_type)
+    : Op{{data, k}}
+    , m_axis{axis}
+    , m_normalized_axis{0}
+    , m_mode{mode_from_string(mode)}
+    , m_sort{sort_type_from_string(sort)}
```

Comment:
You can use this in the constructor instead https://github.com/NervanaSystems/ngraph/pull/4432/files#diff-b358c2a0e091411efef86dea664fb6cdR37 since you have a template specialization for the enum already

Reply:
Done